package.json

```diff
@@ -103,13 +103,13 @@
     "jest-environment-jsdom": "^29.7.0",
     "react-test-renderer": "^18.2.0",
     "style-loader": "^3.3.4",
-    "webpack": "^5.90.1",
+    "webpack": "^5.93.0",
```

Comment:
Is it needed to update the webpack? This is a development and packing tool, if this is not necessary stay to the current version as PR is unit tests related.

Reply:
no need, it happened on my machine because there was some webpack error while I was trying to launch the app, I have restored the webpack version.